app/_src/guides/excluding-traffic-from-transparent-proxy-redirection.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Excluding Traffic from Transparent Proxy
```

Comment:
Should we use some concrete piece of software that the users had problems with before and just show this:

1. the app does not work / start
2. we add excludes
3. the app works

? Would that be more concrete and relateable?